From ae0e151a224ad01870be570d6e8587e3e1c6dde9 Mon Sep 17 00:00:00 2001
From: Rui Miguel Silva <rui.silva@linaro.org>
Date: Tue, 14 Sep 2021 10:46:49 +0100
Subject: [PATCH] arm: corstone1000: enable uefi secure boot

Upstream-Status: Pending [Not submitted to upstream yet]
Signed-off-by: Arpita S.K <arpita.s.k@arm.com>

To make it possible to have a secure way to execute UEFI images
enable UEFI secure boot support and by inherit the
cryptographic functionalities.

Signed-off-by: Rui Miguel Silva <rui.silva@linaro.org>

---
 configs/corstone1000_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/corstone1000_defconfig b/configs/corstone1000_defconfig
index 46ce1780c4b4..fc3a9a21f7ae 100644
--- a/configs/corstone1000_defconfig
+++ b/configs/corstone1000_defconfig
@@ -42,6 +42,7 @@ CONFIG_DM_SERIAL=y
 CONFIG_USB=y
 CONFIG_USB_ISP1760=y
 CONFIG_USB_STORAGE=y
+CONFIG_EFI_SECURE_BOOT=y
 CONFIG_DM_RTC=y
 CONFIG_CMD_RTC=y
 CONFIG_EFI_GET_TIME=y
